plugin-core/src/main/java/appland/actions/OpenRecentAppMapAction.java

```diff
@@ -1,20 +1,20 @@
 package appland.actions;
 
 import appland.AppMapBundle;
+import appland.index.AppMapSearchScopes;
 import com.intellij.openapi.actionSystem.AnAction;
 import com.intellij.openapi.actionSystem.AnActionEvent;
 import com.intellij.openapi.diagnostic.Logger;
 import com.intellij.openapi.fileEditor.FileEditorManager;
 import com.intellij.openapi.ui.Messages;
 import com.intellij.openapi.vfs.VirtualFile;
 import com.intellij.psi.search.FilenameIndex;
-import com.intellij.psi.search.ProjectScope;
 import org.jetbrains.annotations.NotNull;
 
 /**
  * Locates the most recently modified .appmap.json file in the project and opens it.
  * This needs an index and thus can't run when DumbMode is active.
- *
+ * <p>
```

Comment:
Was this intentional?

Reply:
This was added by reformatting the file, so it‘s not strictly needed but would be added again the next time the code is formatted.